ember/app/models/development.js

```diff
@@ -79,8 +79,10 @@ export default class extends DS.Model {
   @computed('address', 'municipal', 'state', 'zipCode')
   get fullAddress() {
     const props = this.getProperties('address', 'municipal', 'state', 'zipCode');
-
-    return `${props.address}, ${props.municipal}, ${props.state} ${props.zipCode}`;
+    const address = props.address ? props.address : 'Unknown Address';
+    const municipal = props.municipal ? props.municipal : 'Unknown City';
+    const zipCode = props.zipCode ? props.zipCode : '';
```

Comment:
Use conditional assignment instead of ternary when the condition is existential:

```js
const zipCode = props.zipCode || '';
```

Reply:
This goes for the other two lines above this one.